sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/servicebus/AzureServiceBusMessagingAutoConfiguration.java

```diff
@@ -142,8 +166,9 @@ AzureMessageConverter<ServiceBusReceivedMessage, ServiceBusMessage> serviceBusMe
         @ConditionalOnBean(ServiceBusProducerFactory.class)
         ServiceBusTemplate serviceBusTemplate(AzureServiceBusProperties properties,
                                               ServiceBusProducerFactory senderClientFactory,
```

Comment:
Please help update this `senderClientFactory` vairable name to `producerFactory`